code/lib/addons/package.json

```diff
@@ -51,6 +51,9 @@
     "@storybook/theming": "7.0.0-alpha.34",
     "global": "^4.4.0"
   },
+  "devDependencies": {
+    "@types/webpack-env": "^1.16.0"
```

Comment:
Why?

Reply:
I think there's a bug with NX where it builds dependents without the --optimized flag

That might be the cause of a few issues... one of those issues I got fixed by adding this, but it's not really what i want to do..

I've reached out to Katarina already

Reply:
Can you explain a little more why that leads to needing to add this? I'm not really following..